<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="fileNameEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>SeoulNamsan</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Enter the name of your file here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="entryEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>SeoulNamsan</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>..or enter the content manually here</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="randomTypeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>SeoulNamsan</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Source of Randomness</string>
       </property>
       <widget class="QRadioButton" name="randomOrgRadio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>random.org</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="mersenneRadio">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Mersenne twister</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>SeoulNamsan</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="entryList">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>256</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseListButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>256</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SeoulNamsan</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SeoulNamsan</family>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: blue;</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
